caffe2/python/operator_test/hardtanh_op_test.py

```diff
@@ -0,0 +1,97 @@
+fro __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from caffe2.python import core
+from hypothesis import given
+import hypothesis.strategies as st
+import caffe2.python.hypothesis_test_util as hu
+import numpy as np
+
+import unittest
+
+
+class TestHardtanh(hu.HypothesisTestCase):
+
+    @given(X=hu.tensor(),
+           engine=st.sampled_from(["", "CUDNN"]),
```

Comment:
You didn't implement CUDNN version of this op, so there is no need to test engine with CUDNN